.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Module",
+            "type": "python",
+            "request": "launch",
+            "module": "uvicorn",
+            "args": [
+                "main:server_app",
+                "--reload"
+            ],
+            "env": {
+                "LOGLEVEL": "DEBUG",
+                "PREFIX": "dev-"
+            },
+            "console": "integratedTerminal",
+            "cwd": "${workspaceFolder}/app"
+        }
+    ]
+}
```

Comment:
Did you mean to commit the debugger configuration file? It might be intended, so I just want to make sure in case it was not.

Reply:
I think it should be made available so that we can use it in the Codespaces without fiddling around

Reply:
Sounds good! Thank you for answering :-)